runtime/types.ts

```diff
@@ -28,11 +28,13 @@ export enum MetadataFormulaType {
 export interface StandardFormulaSpecification {
   type: FormulaType.Standard;
   formulaName: string;
+  chainedCommand?: FormulaSpecification;
```

Comment:
So this file is actually part of the runtime that's used in production. I wouldn't want to change it just for the CLI simulator, I have some suggestions below.